OpenRA.Mods.Cnc/Traits/PoisonedByTiberium.cs

```diff
@@ -49,7 +49,7 @@ public void Tick(Actor self)
 			if (r == null || !Info.Resources.Contains(r.Info.Name))
 				return;
 
-			Info.WeaponInfo.Impact(Target.FromActor(self), self.World.WorldActor, Enumerable.Empty<int>());
+			Info.WeaponInfo.Impact(Target.FromPos(self.CenterPosition), self.World.WorldActor, Enumerable.Empty<int>());
```

Comment:
That will only happen if the actor is already dead, so you can check and bail out inside DoImpact instead.